tasks/skylab/MergeSortBam.wdl

```diff
@@ -57,3 +57,6 @@ task MergeSortBamFiles {
     File output_bam = output_bam_filename
   }
 }
+
```

Comment:
why?

Reply:
We have a new method of merging and a new docker for taking in the output of each scattered job.

Reply:
no i think she just meant the additional space!